docs/build.md

```diff
@@ -0,0 +1,136 @@
+# vue build
+
+`vue build` command gives you a zero-configuration development setup, install once and build everywhere.
+
+## Features
+
+- **Not a boilerplate**: run a single command to develop your app
+- **Out of the box**: ES2015, single-file component with hot reloading and custom CSS preprocessors
+- **Customizable**: populate a `~/.vue/webpack.config.js` for custom webpack config
+- **Single-file component mode**: just run `vue build Component.vue` and boom!
```

Comment:
well, it depends on what the command do. It could be _and test it out in the browser_ or _and distribute it_